### PR TITLE
Fix crash when showing  speaker in SearchActivity

### PR DESCRIPTION
**Pull request by takahirom**
## Issue
- Fix crash when showing  speaker in SearchActivity

## Overview (Required)

- VectorDrawable pathdata is empty in ic_speaker_12_vector.xml
- ic_speaker_12dp_vector.xml is not used.
- So I move ic_speaker_12dp_vector.xml to ic_speaker_12_vector.xml

```
02-06 21:01:54.329 32376 32376 E AndroidRuntime: Caused by: android.content.res.Resources$NotFoundException: File res/drawable/ic_speaker_12_vector.xml from drawable resource ID #0x7f020073
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.content.res.ResourcesImpl.loadDrawableForCookie(ResourcesImpl.java:725)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.content.res.ResourcesImpl.loadDrawable(ResourcesImpl.java:571)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.content.res.Resources.getDrawable(Resources.java:771)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.content.Context.getDrawable(Context.java:525)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v4.content.ContextCompatApi21.getDrawable(ContextCompatApi21.java:30)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v4.content.ContextCompat.getDrawable(ContextCompat.java:372)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at io.github.droidkaigi.confsched2017.view.helper.DataBindingHelper.setSearchResultIcon(DataBindingHelper.java:206)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at io.github.droidkaigi.confsched2017.databinding.ViewSearchResultBinding.executeBindings(ViewSearchResultBinding.java:136)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.databinding.ViewDataBinding.executePendingBindings(ViewDataBinding.java:355)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at io.github.droidkaigi.confsched2017.view.fragment.SearchFragment$SearchResultsAdapter.onBindViewHolder(SearchFragment.java:181)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at io.github.droidkaigi.confsched2017.view.fragment.SearchFragment$SearchResultsAdapter.onBindViewHolder(SearchFragment.java:148)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:6308)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:6341)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:5287)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:5550)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5392)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5388)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2149)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1533)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1496)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:593)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:3535)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.RecyclerView.onMeasure(RecyclerView.java:2979)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.View.measure(View.java:19857)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:715)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:461)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.View.measure(View.java:19857)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6083)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.View.measure(View.java:19857)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:715)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:461)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.View.measure(View.java:19857)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6083)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.support.v7.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:139)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.View.measure(View.java:19857)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6083)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.View.measure(View.java:19857)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6083)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.View.measure(View.java:19857)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6083)
02-06 21:01:54.329 32376 32376 E AndroidRuntime: 	at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1464)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.widget.LinearLayout.measureVertical(LinearLayout.java:758)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.widget.LinearLayout.onMeasure(LinearLayout.java:640)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.View.measure(View.java:19857)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:6083)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.widget.FrameLayout.onMeasure(FrameLayout.java:185)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at com.android.internal.policy.DecorView.onMeasure(DecorView.java:689)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.View.measure(View.java:19857)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:2275)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1366)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1619)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1254)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6337)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:874)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.Choreographer.doCallbacks(Choreographer.java:686)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.Choreographer.doFrame(Choreographer.java:621)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:860)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:751)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6119)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: Caused by: java.lang.IllegalArgumentException: Path string cannot be empty.
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.util.PathParser.nCreatePathDataFromString(Native Method)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.util.PathParser.-wrap1(PathParser.java)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.util.PathParser$PathData.<init>(PathParser.java:74)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1914)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.graphics.drawable.VectorDrawable$VFullPath.inflate(VectorDrawable.java:1865)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.graphics.drawable.VectorDrawable.inflateChildElements(VectorDrawable.java:758)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.graphics.drawable.VectorDrawable.inflate(VectorDrawable.java:659)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.graphics.drawable.DrawableInflater.inflateFromXml(DrawableInflater.java:130)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1227)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.graphics.drawable.Drawable.createFromXml(Drawable.java:1200)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	at android.content.res.ResourcesImpl.loadDrawableForCookie(ResourcesImpl.java:715)
02-06 21:01:54.330 32376 32376 E AndroidRuntime: 	... 67 more
02-06 21:01:54.332   880 16661 W ActivityManager:   Force finishing activity io.github.droidkaigi.confsched2017.debug/io.github.droidkaigi.confsched2017.view.activity.SearchActivity
02-06 21:01:54.660   880 16661 W ActivityManager:   Force finishing activity io.github.droidkaigi.confsched2017.debug/io.github.droidkaigi.confsched2017.view.activity.SearchActivity
02-06 21:01:54.719   880 16661 W ActivityManager:   Force finishing activity io.github.droidkaigi.confsched2017.debug/io.github.droidkaigi.confsched2017.view.activity.MainActivity
02-06 21:01:54.743   880 16661 I ActivityManager: Killing 32376:io.github.droidkaigi.confsched2017.debug/u0a277 (adj 199): crash

```

### CODE REVIEW

**Comment by konifar** [comment]
🙇 Good catch!!

File: app/src/main/res/drawable/ic_speaker_12_vector.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
         android:viewportWidth="24.0"
         >
     <path
-            android:fillColor="@color/grey600"
-            android:pathData=""
+            android:fillColor="@color/grey500"
+            android:pathData="M12,12c2.21,0 4,-1.79 4,-4s-1.79,-4 -4,-4 -4,1.79 -4,4 1.79,4 4,4zM12,14c-2.67,0 -8,1.34 -8,4v2h16v-2c0,-2.66 -5.33,-4 -8,-4z"
```
Comment (konifar) [inline app/src/main/res/drawable/ic_speaker_12_vector.xml]: 🙇 OMG. It's my mistake...
Thanks for your fix!

**Comment by konifar** [comment]
LGTM!! 💯

**Comment by takahirom** [comment]
Thanks!!! 👍

**Pull request merged**